modules/billing/variables.tf

```diff
@@ -0,0 +1,133 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# These variables must be set when using this module.
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "billing_account" {
+  type        = string
+  description = "(Required) ID of the billing account to set a budget on."
+}
+
+variable "threshold_rules" {
+  type = any
+  # type = list(object({
+  #   # (Required) Send an alert when this threshold is exceeded. This is a 1.0-based percentage, so 0.5 = 50%. Must be >= 0.
+  #   threshold_percent = number
+  #   # Optional) The type of basis used to determine if spend has passed the threshold. Default value is `CURRENT_SPEND`. Possible values are `CURRENT_SPEND` and `FORECASTED_SPEND`.
+  #   spend_basis = optional(string)
+  # }))
+  description = "(Required) Rules that trigger alerts (notifications of thresholds being crossed) when spend exceeds the specified percentages of the budget."
+  default = [0.2,0.4,0.6,0.7,0.8,0.9,1]
```

Comment:
I think that this `default` value can be removed... If we, for example, forget to set the correct values it would be better to have an error rather than it to be swallowed because there is a default value present.